deploy/kubectl_helpers/restore_local_db.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -x -e
```

Comment:
You may want a `-u` here too, so that the script exits if any of the variables are undefined. The default will be to continue running the commands with empty strings in place of the empty variables. These may very well result in benign errors that are caught by the -e, but I feel like it's best to bail before trying to execute these gcloud and psql commands with partial inputs.